consensus/src/epoch_manager.rs

```diff
@@ -308,14 +308,17 @@ impl EpochManager {
             Ok(()) => { /* Expected */ }
             Err(Error::ValidatorKeyNotFound(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "Unable to find the validator consensus key during initialize",
+                    "Unable to find the validator consensus key during initialize() and will not participate in consensus.\
```

Comment:
The error is simply "Unable to initialize safety rules" We can consider the second part on "not participate in consensus" see my next comment.